core/micro-services/workflow-operator/src/main/scala/edu/uci/ics/amber/operator/projection/AttributeUnit.java

```diff
@@ -19,7 +20,10 @@ public class AttributeUnit{
     @JsonPropertyDescription("Renamed attribute name")
     private String alias;
 
-    AttributeUnit(String attributeName, String alias) {
+    @JsonCreator
+    public AttributeUnit(
+            @JsonProperty("originalAttribute") String attributeName,
+            @JsonProperty("alias") String alias) {
```

Comment:
why those changes?

Reply:
Without this change, there will be error during deserializing the json to AttributeUnit of the Projection operator:

Cannot construct instance of `edu.uci.ics.amber.operator.projection.AttributeUnit` (no Creators, like default constructor, exist): cannot deserialize from Object value (no delegate- or property-based Creator)

Reply:
we need to understand the reason. we can postpone the analysis, but please keep it a todo or track the task some where.

Reply:
added